REST_API_DOCS.md

```
@@ -0,0 +1,1302 @@
+| 파라미터 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| page | integer | X | 페이지 번호(1페이지 부터 시작) |
+| limit | integer | X | 페이지당 게시글 개수 |
+
+**Response (200 OK)**
+
+```json
+{
+  "status": "success",
+  "data": [
+	    {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)